lib/src/benchmark_base.dart

```diff
@@ -62,3 +62,64 @@ class BenchmarkBase {
     emitter.emit(name, measure());
   }
 }
+
+class AsyncBenchmarkBase {
```

Comment:
Put this in its own async_benchmark_base.dart file.

Reply:
Done